src/database.ts

```diff
@@ -1874,34 +1890,53 @@ class Database extends common.GrpcServiceObject {
    * const instance = spanner.instance('my-instance');
    * const database = instance.database('my-database');
    * const backupName = 'projects/my-project/instances/my-instance/backups/my-backup';
-   * const [, restoreOperation] = await database.restore(backupName);
+   * const [, restoreOperation] = await database.restore(
+   *   backupName,
+   *   {
+   *     encryptionConfig: {
```

Comment:
Depending on how often we think that users will use encrypted backups, I would consider leaving the encryption information out of this example. As it is now, it at first hand seems to be something that is required for all restore operations, while it is only required if you actually have an encrypted backup.

Reply:
Yeah, I think it's reasonable to remove this as it's not something you have to specify by default. Sometimes we have multiple examples so I may consider adding a separate one.

BTW, for restoring with an encryption key, it's not a requirement to specify the encryption key if the backup was encrypted. If the backup was encrypted, then the restored database is automatically encrypted with the same key as the backup. You only have to specify the key here if you want to encrypt it with a different key.